6_0_upgrade.md

```diff
@@ -11,19 +11,22 @@ straightforward.
 3. Rename `config/webpack` to `config/webpack_old`
 4. Rename `config/webpacker.yml` to `config/webpacker_old.yml`
 5. Uninstall the current version of `webpack-dev-server`: `yarn remove webpack-dev-server`
-6. Upgrade webpacker
+6. Upgrade webpacker to the latest prelease
+
+- [Check the releases page to verify the latest version](https://github.com/rails/webpacker/releases)
+- Make sure to install identical version numbers of webpacker gem and `@rails/webpacker` npm package
 
   ```ruby
   # Gemfile
-  gem 'webpacker', '~> 6.0.0.pre.2'
+  gem 'webpacker', '6.0.0.beta.6'
   ```
 
   ```bash
   bundle install
   ```
 
   ```bash
-  yarn add @rails/webpacker@next
+  yarn add @rails/webpacker@6.0.0.beta.6
```

Comment:
```suggestion
  yarn add @rails/webpacker@6.0.0-beta.6
```

The yarn package has a `-` instead of a `.` between `6.0.0` and `beta.6`.

Reply:
Good catch!